src/run_solver.jl

```diff
@@ -124,7 +126,7 @@ function solve_problems(
             end
           end
 
-          @info log_header(colstats, types[col_idx], hdr_override = info_hdr_override)
+          @info  "Solver"    * " " *  log_header(colstats, types[col_idx], hdr_override = info_hdr_override)
```

Comment:
The following account for the fact the name of the solver can be longer than the word `Solver`
```suggestion
          @info "Solver" * (" ")^max(length(solver_name) - 5, 1) * log_header(colstats, types[col_idx], hdr_override = info_hdr_override)
```

Reply:
I don't think we need the string "Solver" here. Just the solver name, and it should always be present, i.e., I don't think it should be a kwarg that defaults to the empty string.

Reply:
@dpo Ok, that works for me too.
If we add a new argument to the `solve_problem` with the name, then the tests need to be changed.
I will update my comments, then.

Reply:
@DoctorDro the package has unit tests on several machines that were failing with your modification that's why you received this. You can run them on your machine using `julia> ] test` for instance.

Please, mark the conversations as resolved only when changes have been made to the code.

The commit suggestion made by Github is to commit on your branch, it's still not on the repo. It shows when we add comments with code suggestions to help you navigate. "Add suggestion to batch" allow to commit more comments than one by one.